niceman/distributions/tests/test_debian.py

```diff
@@ -15,7 +15,10 @@
 
 import mock
 
+from ...tests.utils import skip_if_on_osx
 
+
+@skip_if_on_osx
```

Comment:
should better be 

`@skip_if(not on_debian)`

this way we could run tests on centos etc